_opensearch/supported-field-types/date.md

```diff
@@ -209,3 +209,127 @@ GET testindex/_search
   }
 }
 ```
+
+## Date math
+
+The date field type supports date math to specify duration in queries. For example, the `gt`, `gte`, `lt`, and `lte` parameters in [range queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/term/#range-query) and the `from` and `to` parameters in [date range aggregations]({{site.url}}{{site.baseurl}}/opensearch/bucket-agg/#range-date_range-ip_range) accept date math expressions.
+
+A date math expression contains a fixed date, optionally followed by one or more mathematical expressions. The fixed date may be either `now` (current date/time in milliseconds since the epoch) or a string ending with `||` that specifies a date (for example, `2022-05-18||`). The date must be in the `strict_date_optional_time||epoch_millis` format.
+
+Date math supports the following mathematical operators:
+
+Operator | Description | Example
+:--- | :--- | :---
+`+` | Addition | `+1M`: Add one month
+`-` | Subtraction | `-1y`: Subtract one year
+`/` | Rounding down | `/h`: Round to the beginning of the hour
+
+Date math supports the following time units:
```

Comment:
Is this supposed to be formatted as a table, or a list? If so, replace the colon with a period ("units.")

Reply:
Same for subsequent sections where this may apply.

Reply:
Not a table; kind of a list. This matches the previous section's unit listing.